### ISSUE TITLE
Основное и дополнительное

### ISSUE THREAD

**Issue by pavelvasev** [Mitricho/DocsiteMuse#2]
Посередине между дизайном и панелью параметров идет вертикальный разделитель с иконками. Предложение поменять порядок иконок в нем. Сначала основные, ими я считаю прокрутку вдоль панели параметров (общие настройки. баннер, и т..п.). Затем какой-то ясный визуальный разделитель. И затем доп опции - редактор палитры, о программе, и т.п.

Основным блоком я считаю то потому что это основное что надо в работе, чаще всего используется.

**Comment by Mitricho** [comment]
Не знаю. Думаю.

**Comment by Mitricho** [comment]
Пока поправил так: работа с проектом | навигация по простыне | дополнительное

**Comment by pavelvasev** [comment]
Да, вижу, стало лучше. Мое мнение что можно и работу с проектом задвинуть, после навигации по простыне, т.к. работа с проектом это редкое явление, а навигация 95% времени.

**Comment by pavelvasev** [comment]
я бя предложил попробовать их подсвечивать, таки, посмотреть что получается. пока почему-то мозг их не выхватывает, и не предлагает ими пользоваться (ну мой).. - итого вся эта панель пока воспринимается им как не-нужная.
